Code/EnginePlugins/RecastPlugin/NavMeshBuilder/NavMeshBuilder.cpp

```diff
@@ -1,15 +1,18 @@
 #include <RecastPlugin/RecastPluginPCH.h>
 
-#include <Core/Utils/WorldGeoExtractionUtil.h>
 #include <Core/World/World.h>
 #include <Foundation/Time/Stopwatch.h>
 #include <Foundation/Types/ScopeExit.h>
+#include <Foundation/Utilities/GraphicsUtils.h>
 #include <Foundation/Utilities/Progress.h>
 #include <Recast/DetourNavMesh.h>
 #include <Recast/DetourNavMeshBuilder.h>
 #include <Recast/Recast.h>
 #include <RecastPlugin/NavMeshBuilder/NavMeshBuilder.h>
 #include <RecastPlugin/Resources/RecastNavMeshResource.h>
+#include <RendererCore/Meshes/CpuMeshResource.h>
+#include <RendererCore/Meshes/MeshBufferUtils.h>
+#include <RendererCore/Utils/WorldGeoExtractionUtil.h>
```

Comment:
Did this plugin already have a link dependency on RendererCore ?

Reply:
Yes, it is already used for the navmesh visualization.

Reply:
Would it make sense to move Mesh stuff to core since meshes are not just for rendering?